### ISSUE TITLE
学习率不降低反而增加，为什么？

### ISSUE THREAD

**Issue by happybear1015** [PaddlePaddle/PaddleClas#2776]
关键配置信息如下：
# model architecture
Arch:
  name: ResNet101_vd
  class_num: 2

# loss function config for traing/eval process
Loss:
  Train:
    - CELoss:
        weight: 1.0
        epsilon: 0.1
  Eval:
    - CELoss:
        weight: 1.0

Optimizer:
  name: Momentum
  momentum: 0.9
  lr:
    name: Cosine
    learning_rate: 0.1
  regularizer:
    name: 'L2'
    coeff: 0.0001

**Comment by happybear1015** [comment]
![lr(CosineAnnealingDecay)](https://user-images.githubusercontent.com/66996161/236358861-4b6ecb99-b4b8-40d4-9856-3a2e34cbd5f5.png)

**Comment by zengshao0622** [comment]
尝试过调整学习率等参数了吗？

**Comment by happybear1015** [comment]
是这样的，训练到120epoch时候，学习率就开始增加了（从0-120期间，学习率都是正常在下降）

**Comment by zengshao0622** [comment]
可以提供一下最小复现代码吗？

**Comment by happybear1015** [comment]
数据量比较庞大

**Comment by happybear1015** [comment]
你需要训练日志吗？

**Comment by zengshao0622** [comment]
我需要复现学习率异常问题的demo代码定位一下问题呢

**Comment by happybear1015** [comment]
您可以远程操作一下嘛，这样快



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年5月5日(周五) 中午11:20
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PaddlePaddle/PaddleClas] 学习率不降低反而增加，为什么？ (Issue #2776)




 
我需要复现学习率异常问题的demo代码定位一下问题呢

**Comment by zengshao0622** [comment]
还是辛苦你整理一个最小复现代码吧～
